hack/vendor.sh

```diff
@@ -55,7 +55,7 @@ clone hg code.google.com/p/go.net 84a4013f96e0
 clone hg code.google.com/p/gosqlite 74691fb6f837
 
 #get libnetwork packages
-clone git github.com/docker/libnetwork 005bc475ee49a36ef2ad9c112d1b5ccdaba277d4
+clone git github.com/docker/libnetwork f72ad20491e8c46d9664da3f32a0eddb301e7c8d
```

Comment:
@duglin in order to bump a version of a dependency, we change it in `hack/vendor.sh` and run the script locally which will pull in all the required changes into the `vendor` dir.

@runcom not necessarily always. especially if a change in dependent package causes any API or semantic changes, then the corresponding change in docker/docker must be done in the same commit.